cmd/user-cluster-controller-manager/main.go

```diff
@@ -112,6 +114,8 @@ func main() {
 	flag.IntVar(&runOp.opaWebhookTimeout, "opa-webhook-timeout", 3, "Timeout for OPA Integration validating webhook, in seconds")
 	flag.BoolVar(&runOp.useSSHKeyAgent, "enable-ssh-key-agent", false, "Enable UserSSHKeyAgent integration in user cluster")
 	flag.StringVar(&runOp.caBundleFile, "ca-bundle", "", "The path to the cluster's CA bundle (PEM-encoded).")
+	flag.IntVar(&runOp.mlaGatewayPort, "mla-gateway-port", 0, "The port of MLA(Monitoring, Logging, and Alerting) gateway.")
```

Comment:
This is not possible, basically this `mlaGateway` will be the NodePort which will be assigned to `MLAGateway` service in seed cluster, and it will be assigned by Kubernetes, just like what we did for `openvpnServerPort`